src/compass.js

```diff
@@ -46,6 +47,9 @@ function clearCompass() {
 
 function draw(what) {
   console.log("Call: Compass.draw");
+  compass.backgroundColor(Themes.currentTheme().textColour);
```

Comment:
I think personally, I like the idea of another method. Maybe, "reinit"? Or call it "configureUI" and call it from each init method and when settings are updated?